src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.pkgproj

```diff
@@ -0,0 +1,44 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <GeneratePackage>true</GeneratePackage>
+    <IncludeSymbolsInPackage>true</IncludeSymbolsInPackage>
+    <PackageDescription>Provides a native AOT compiler and runtime for .NET</PackageDescription>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- For now, send symbols package into a dummy directory to avoid conflicts with dotnet/runtime and
+         to avoid issues with objwriter not having the rigth ELF version resource -->
+    <SymbolPackageOutputPath>$([MSBuild]::NormalizeDirectory('$(ArtifactsPackagesDir)', 'Ignored'))</SymbolPackageOutputPath>
```

Comment:
If we send them into `NonShipping` do them also get uploaded to symbol server? @mmitche do you know?

Reply:
I'll just try this without the SymbolPackageOutputPath - the concern about dotnet/runtime conflict in the comment no longer applies and the objwriter package is now built with Arcade's help.

Reply:
I **think** they do. But not positive.

Reply:
Might just do a quick official test (assign the build to General Testing) and see what happens.